### PR TITLE
Coverage plugin changes

### PR DESCRIPTION
**Pull request by AlexeySanko**
PyBuilder has changes into `coverage` plugin which weren't included into documentation

### CODE REVIEW

File: documentation/plugins.md [hunk 1]
```diff
@@ -92,26 +92,40 @@ Use the ```python.coverage``` module to activate coverage.
...
-
+  
   <tr>
-    <td>coverage_reload_modules</td>
+    <td>coverage_allow_non_imported_modules</td>
     <td>bool</td>
     <td>True</td>
-    <td>Reload modules during coverage to also count definition lines.</td>
-  </tr>
+    <td>Allow modules which was not imported by the covered tests.</td>
```
Comment (arcivanov) [inline documentation/plugins.md]: that were not

File: documentation/plugins.md [hunk 2]
```diff
@@ -122,6 +136,8 @@ Use the ```python.coverage``` module to activate coverage.
   </tr>
 </table>
 
+Pay attention that properties `coverage_reload_modules` and `coverage_fork` are deprecated and will not be used.
```
Comment (arcivanov) [inline documentation/plugins.md]: Please note that

File: documentation/plugins.md (hunk 1 above)
Comment (AlexeySanko) [reply documentation/plugins.md]: Thanks. Fixed

File: documentation/plugins.md (hunk 2 above)
Comment (AlexeySanko) [reply documentation/plugins.md]: Thanks. Fixed

**Pull request merged**